dask/dataframe/core.py

```diff
@@ -1368,7 +1370,7 @@ def to_sql(
         return to_sql(
             self,
             name=name,
-            uri=uri,
+            con=con,
```

Comment:
I wouldn't rename this, you will likely break people's code. Since it can be either an engine or a URI, either name would do, but "conn" is odd.

Reply:
this function doesn't exist in Dask yet; it's added by https://github.com/dask/dask/pull/6038, which this PR is a further PR against, so I don't think it's going to break anyones' code.

[`con` is what Pandas uses for this](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.to_sql.html)

Reply:
pandas says it accepts a "connection" object, hence the name (or an engine). We accept an Engine or text, so I think it's problematic. The breakage would be in the rename in read_sql_table, which ought to match this name, ideally

Reply:
good points!

I name it `uri` in both functions now, and have updated the docs